wiki/Converting_sequence_files.md

```diff
@@ -28,14 +28,10 @@ directory):
 ``` python
 from Bio import SeqIO
 
-input_handle = open("cor6_6.gb", "rU")
-output_handle = open("cor6_6.fasta", "w")
-
-sequences = SeqIO.parse(input_handle, "genbank")
-count = SeqIO.write(sequences, output_handle, "fasta")
-
-output_handle.close()
-input_handle.close()
+with open("cor6_6.gb", "rU") as input_handle, open("cor6_6.fasta", "w") as output_handle:
```

Comment:
You can use the double with statement from Python 2.7 onwards, which is what Biopython 1.69 will require.

However, this does make the line rather long, so perhaps two nested with statements is better here?